requirements/base.txt

```diff
@@ -34,10 +34,10 @@ itypes==1.1.0             # via coreapi
 jinja2==2.10.1            # via coreschema
 markupsafe==1.1.1         # via jinja2
 mysqlclient==1.4.2.post1
-newrelic==4.20.1.121      # via edx-django-utils
-oauthlib==3.0.2           # via requests-oauthlib, social-auth-core
+newrelic==5.0.0.124       # via edx-django-utils
```

Comment:
edx-platform had an issue with this version of new relic: https://github.com/edx/edx-platform/pull/21312